lib/api/src/modules/versions.ts

```diff
@@ -33,7 +33,7 @@ export interface SubState {
 const getVersionCheckData = memoize(1)(
   (): Versions => {
     try {
-      return JSON.parse(VERSIONCHECK).data as Versions;
+      return { ...(JSON.parse(VERSIONCHECK).data || {}) };
```

Comment:
@ndelangen why s the `{ ... }` needed?

Reply:
to ensure the returned value is an object.

I'm unsure how it happened but in #8706 if window.VERSION is parsable, and accessing data didn't throw an error, it would be returned. (it could be undefined)

Now I made sure data is defined, and is an object.

If something is attempted to be spread, that cannot be spread, it will throw, and the catch-clause will return an empty object.